src/main/java/me/totalfreedom/totalfreedommod/command/Command_smite.java

```diff
@@ -38,22 +39,23 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             return true;
         }
 
-        smite(player, reason);
+        smite(player, sender, reason);
         return true;
     }
 
-    public static void smite(Player player)
+    public static void smite(Player player, CommandSender sender)
```

Comment:
No, https://gyazo.com/ea18e6f50ae83a44d274e8cf5380cfb9 It causes errors without CommandSender sender...